bitnami/mysql/templates/primary/statefulset.yaml

```diff
@@ -186,6 +186,10 @@ spec:
           ports:
             - name: mysql
               containerPort: 3306
+            {{- if .Values.mysqlx.port.enabled }}
```

Comment:
Does it make sense for replicas also?

Reply:
I am not sure if replicas can also listen on mysqlx protocol.

Reply:
have made the necessary changes.